certora/harness/MetaMorphoHarness.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity 0.8.21;
+
+import "../munged/MetaMorpho.sol";
+
+contract MetaMorphoHarness is MetaMorpho {
```

Comment:
is not yet IMetaMorhpoHarness right ?

Reply:
There is no IMetaMorphoHarness, because we don't really need an interface for using a contract with Certora

Reply:
ah yes sorry I messed up. So why do we need this contract?

Reply:
This is not really needed in this PR, but it will be useful at some point so I thought we could start with it. For example, in #385 we need the harness